src/rightClick.js

```diff
@@ -0,0 +1,70 @@
+// Register the Plugin when installed
+chrome.runtime.onInstalled.addListener(() => {
+	// TODO: GET INSTANCE ID FROM LAMBDA
+	chrome.storage.local.set({ instanceID: "xxxxxxxx-xxxx-Mxxx-Nxxx-xxxxxxxxxxxx" });
+	//chrome.browserAction.setIcon({
+	//	path: {
+	//		"128": 'images/128_CCC_GREEN.png'
+	//	}
+	//})
+	mapMenus();
+});
+
+// Load available infractions when a window is opened
+chrome.windows.onCreated.addListener(async () => {
+	await mapMenus();
+});
+
+//https://developer.chrome.com/docs/extensions/mv3/migrating_to_service_workers/#state
+// Listen for when we are clicked 
+chrome.contextMenus.onClicked.addListener(async (info, tab) => {
+	//handle context menu actions
+	//await chrome.browserAction.setIcon('images/128_CCC_BLUE.png')
+	const htmlElement = await chrome.storage.local.get(["htmlElement", "instanceID"], storage => {
+		const infraction = {
+			reporter: storage.instanceID,
+			url: info.pageUrl,
+			type: info.menuItemId,
+			content: storage.htmlElement
+		};
+		console.log(infraction)
+		// TODO: SEND THIS TO THE INFRACTION LOGGING API
+		//await fetch(`https://cyber-citizenship-coefficient.com/infraction`, 
```

Comment:
@ajaxx 
Once we have your [PR](https://github.com/Cyberspace-Citizenship-Coefficient/AWS-Lambdas/pull/17) done we can send data in